### PR TITLE
feat: Requirements and minimum hold for drops

### LINKED ISSUE #9456: Requirements and minimum hold for drops
## auto-teleport and confirm modal

for some reason, our confirmation dialog for paid mint is gone
- in this dialog/modal, there was also an option to auto-teleport 

since we operate on an asset hub, it's important to have the option to teleport as most of the users that come don't have funds there

image

- probably without the orange box

## Required balance
- if the required balance = mint price, we don't have to show it as a requirement - the user does not get to the modal where we can teleport, and its confusing 
- i think we agreed internally that having an additional balance requirement on top is not super useful as it's hard to explain.

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #9456

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [ ] My fix has changed **something** on UI;

### CODE REVIEW

**Comment by hassnian** [comment]
> Making it for release? 😄

will the next drop be a holder of ? the issue itself is done,  fixing other bugs related to paid mint that I found , can be done in another pr

**Comment by exezbcz** [comment]
@hassnian nope, just paid

**Comment by exezbcz** [comment]
image
nice 

@prury can you make sure autoteleport is enabled in the modal as well?

**Comment by exezbcz** [comment]
imo there does not have to be the requirement to hold on "assethub" since autoteleport is enabled 

we can IMO remove the requirements and just leave the heading with "open" indicator 

followup for these sections is in 
- #9410

File: components/collection/drop/modal/PaidMint.vue [hunk 1]
```diff
@@ -50,18 +50,24 @@ import { usePreloadMintedNftCover } from './utils'
...
+    modelValue: boolean
+    toMintNft: ToMintNft
+    action: AutoTeleportAction
+    isAllocatingRaffle: boolean
+    minimumFunds: number
+    hasMinimumFunds: boolean
+    hideMinimumFundsWarning: boolean
+    formattedMinimumFunds: string
+    formattedExistentialDeposit: string
+    mintedNft?: DropMintedNft
```
Comment (preschian) [inline components/collection/drop/modal/PaidMint.vue:64]: eslint warning, default value

**Comment by prury** [comment]
how is this already working on canary tho? I'm a bit confused, can someone else confirm?

**Comment by hassnian** [comment]
> how is this already working on canary tho(confirmation dialog)? I'm a bit confused, can someone else confirm? 

You have to try a drop that has 

**holder of + price** requirement 

Then you will see the modal

**Comment by prury** [comment]
> > how is this already working on canary tho(confirmation dialog)? I'm a bit confused, can someone else confirm?
> 
> You have to try a drop that has
> 
> **holder of + price** requirement
> 
> Then you will see the modal

just deleted my comment as i have figured out lol, ty, will try on reflection/complexity

**Comment by prury** [comment]
> 
> @prury can you make sure autoteleport is enabled in the modal as well?

as you said on another comment/follow up, we have to remove requirement, otherwise autoteleport will not fire up

**Comment by yangwao** [comment]
pay 50 usd
testing this from 2km altitude

**Pull request merged**